src/app/components/ResourceHints/index.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
-import { arrayOf, string } from 'prop-types';
+import React, { Fragment } from 'react';
+import { any, objectOf, string } from 'prop-types';
 
-const ResourceHints = ({ assetOrigins }) => (
-  <>
-    {assetOrigins.map(origin => (
-      <link
-        rel="preconnect"
-        key={origin}
-        href={origin}
-        crossOrigin="anonymous"
-      />
-    ))}
-    {assetOrigins.map(origin => (
-      <link rel="dns-prefetch" key={origin} href={origin} />
-    ))}
-  </>
-);
+const ResourceHints = ({ assetOrigins, service }) => {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const serviceConfig = require(`../../lib/config/services/${service}`);
+  const { fonts } = serviceConfig.service.default;
```

Comment:
I think we need to handle services with variants (i.e. not just 'default') here too such as https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/zhongwen.js